sql/log_event_server.cc

```diff
@@ -2705,10 +2706,9 @@ static void check_and_remove_stale_alter(Relay_log_info *rli)
   {
     DBUG_ASSERT(info->state == start_alter_state::REGISTERED);
 
-    sql_print_warning("ALTER query started at %u-%lu-%llu could not "
+    sql_print_warning("ALTER query started at %u-%u-%llu could not "
                       "be completed because of unexpected master server "
-                      "or its binlog change", info->domain_id,
-                      mi->master_id, info->sa_seq_no);
+                      "or its binlog change", PARAM_GTID(info->gtid));
```

Comment:
Maybe real reason I swapped to a nested struct is so I can write this `PARAM_GTID`? 🙃